api/src/pcapi/routes/backoffice/offers/blueprint.py

```diff
@@ -209,13 +206,14 @@
 
 def _get_offer_ids_query(form: forms.InternalSearchForm) -> BaseQuery:
     query = offers_models.Offer.query
-
-    if not forms.GetOfferAdvancedSearchForm.is_search_empty(form.search.data):
+    advanced_search = not forms.GetOfferAdvancedSearchForm.is_search_empty(form.search.data)
```

Comment:
Détail : comme c'est un boolén, je trouve plus clair avec "is_" devant dans ce cas : 
```suggestion
    is_advanced_search = not forms.GetOfferAdvancedSearchForm.is_search_empty(form.search.data)
```